lib/vite-postcss-config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('node:path');
+const fs = require('node:fs');
+const postcssViewportToContainerToggle = require('postcss-viewport-to-container-toggle');
+
+module.exports = async ({
+  assetOptions, util, isPublic = false
+}) => {
+  const postCssConfigPath = path.resolve(process.cwd(), 'postcss.config.js');
+  const importedConfig = isPublic && fs.existsSync(postCssConfigPath)
+    ? await import(postCssConfigPath)
+    : {};
+
+  const {
+    plugins = [],
+    ...postCssConfig
+  } = importedConfig.default || importedConfig;
+
+  if (!Array.isArray(plugins)) {
+    util.error('WARNING: postcss.config.js must export an array of plugins');
+  }
```

Comment:
We force the user to use plugins as arrays like described in the doc, otherwise making the require from here would not work.
Like this:
```javascript
import autoprefixer from 'autoprefixer';
import tailwindcss from 'tailwindcss';

export default {
  plugins: [ autoprefixer(), tailwindcss() ]
};
```

Not like this:
```javascript
export default {
  plugins: {
    autoprefixed: {},
    tailwindcss: {}
  }
};

```

Reply:
Look at my recommendation (HUGE) comment. Yes, this is a great idea and I also think we should throw and not only warn. The reason for that is:
- If a project transitions to our Vite build and already has a postcss config due to internal other use of potscss, the project developers should be aware that there is a conflcit and they are breaking the Apostrophe build pipeline. 
- They can make a decision how to proceed because they know stuff is broken on build time, which is GOOD. 

I also think we need to ping Bob so that he can add important warning boxes in the docs about the whole possible postcss configuration conflicts and requirements - this one can be cited from the Vite docs, link in my previous comment.

Reply:
Both versions are working with the loader now.